content/blog/tdd-2025/index.Rmd

```diff
@@ -0,0 +1,66 @@
+---
+output: hugodown::hugo_document
+
+slug: tdd-2025
+title: Tidyverse developer day 2025
+date: 2025-06-24
+author: Max Kuhn
+description: >
+     Join us in Atlanta for tidyverse developer day on September 19, 2025!
+
+photo:
+  author: George Stagg
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [other] 
+tags: [tidyverse-dev-day]
+---
+
+<!--
+TODO:
+* [ ] Look over / edit the post's title in the yaml
+* [ ] Edit (or delete) the description; note this appears in the Twitter card
+* [ ] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+
+The 2025 Tidy Dev Day will be on September 19th! We'll gather after posit::conf at the TODO from TODO to TODO. 
+
+Tidy Dev Day is an event where developers and open source contributors meet to collaboratively solve issues. You can hone your software engineering skills or work on your first contribution. We will have a slate of existing issues that you can work on with or without the assistance of our developers. 
+
+Here's how [Laura Acion](https://lacion.rbind.io/post/my-first-user/) described the 2019 event:
+
+> I wish there were more of these R-related dev days. Developing events that are friendly, diverse, and safe to advance not only the tidyverse. I see no downsides. Kuddos to the organizers.
+
+Plus...
+
+<div style="text-align: center">
+<img src="GONG.jpg" alt="plot of chunk gong" width="70%" />
+</div>
+
+<br>
+
+## Who should attend
+
+You should attend if: 
```

Comment:
I kind of admire the breezy tone here, but worry that if someone takes this at face value, they might show up effectively needing 1-on-1 help for the entire day. I think getting an R package dev environment set up in advance (or at least trying *very hard* to do so) is basically a pre-req. I would mention the Git/GitHub workshop that @mine-cetinkaya-rundel will run as recommended prep for folks who've never made a PR before.